securedrop/tests/test_unit_integration.py

```diff
@@ -260,6 +260,20 @@ def test_reply_normal(self):
     def test_reply_unicode(self):
         self.helper_test_reply("Teşekkürler", True)
 
+    def test_encoding_robustness_against_env(self):
+        # The LC_ALL locale env var is the preferred env var that
+        # locale.getpreferredencoding() looks for. If it is not defined, the
+        # method looks at other locale env vars.
+        lc_all_orig = os.getenv("LC_ALL") 
+        os.environ["LC_ALL"] = "C"
+        try:
+            self.helper_test_reply('Ivan Krsti\u0107', True)
+        finally:
+            if lc_all_orig:
+                os.environ["LC_ALL"] = lc_all_orig
+            else:
+                os.environ["LC_ALL"] = ""
```

Comment:
Also, this test does not work. It passes even if I comment out the call to `_make_binary_stream` in `crypto_util.encrypt`.